recipes/scikit-lego/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "scikit-lego" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/koaning/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "af41a0803198434ca2ba40dfe726b9c784ced1f02f50a9d3e0a840bc3c08fec8"
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

This Python looks like pure Python code and thus can be noarch
```suggestion
  noarch: python
```